src/main/java/frc/robot/commands/AngleAlgaeCollectorCommand.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.MotorPowerController;
+import frc.robot.Constants.SensorStatus;
+import frc.robot.subsystems.AlgaeCollectionSubsystem;
+
+public class AngleAlgaeCollectorCommand extends Command{
+
+    Timer timeoutTimer = new Timer();
+    Timer atAngleTimer = new Timer();
+    AlgaeCollectionSubsystem algaeCollectionSubsystem;
+    MotorPowerController motorPowerController;
+
+    public AngleAlgaeCollectorCommand(AlgaeCollectionSubsystem algaeCollectionSubsystem) {
+        this.algaeCollectionSubsystem = algaeCollectionSubsystem;
+        addRequirements(algaeCollectionSubsystem);
+        motorPowerController = new MotorPowerController(0.03, 0.3, 0.2, 0.3, 1, SensorStatus.kClimberAngle, 5);
+        // Use addRequirements() here to declare subsystem dependencies.
+
+    }
+
+    @Override
+    public void initialize() {
+        timeoutTimer.reset();
```

Comment:
Isn't this only ran at start? How will these be reset after work is done? Do they need reset in end()